cpu/cc2538/radio/cc2538_rf_getset.c

```diff
@@ -230,6 +230,8 @@ void cc2538_set_state(cc2538_rf_t *dev, netopt_state_t state)
             RFCORE_WAIT_UNTIL(RFCORE->XREG_FSMSTAT0bits.FSM_FFCTRL_STATE > FSM_STATE_RX_CALIBRATION);
             dev->state = NETOPT_STATE_IDLE;
             break;
+        default:
+            break;
```

Comment:
Because this PR introduces a new netopt state for LoRA which is not handled by this switch. Thus the build for cc2538 was raising `warning: enumeration value '[...]'not handled in switch [-Wswitch]` => error.
I can open a dedicated PR if you prefer.